app/models/reports/teammate_review_report.rb

```
@@ -0,0 +1,110 @@
+
+    # -----------------------------------------------------------------------
+    # Reducer 2 — per-reviewer teammate count.
+    # Streams TeamsUser rows for the assignment. Only updates state for
+    # users who are already reviewers (present in shared state).
+    # -----------------------------------------------------------------------
+    class TeamSizeReducer < BaseReport
+      def source
+        TeamsUser.for_assignment(@reportable.id).includes(:user)
+      end
+
+      def state_key_for = ->(teams_user) { teams_user.user_id }
```

RuboCop (1.86.2)

[convention] Pass `&:user_id` as an argument to `lambda` instead of a block.

(Style/SymbolProc)